homeassistant/components/control4/light.py

```diff
@@ -191,6 +113,10 @@ def supported_features(self) -> int:
             return SUPPORT_BRIGHTNESS | SUPPORT_TRANSITION
         return 0
 
+    @property
+    def _is_dimmer(self):
+        return bool("LIGHT_LEVEL" in self.extra_state_attributes)
```

Comment:
We shouldn't access `extra_state_attributes`.  We should access the data that is being used to create it instead.